api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -260,12 +257,43 @@ model PlatformProfile {
 
   /** Specifies whether subnets are pre-attached with an NSG */
   preconfiguredNsgs: boolean;
+}
 
-  /** The id of the disk encryption set to be used for etcd.
-   * Configure this when `etcdEncryption` is set to true
-   * Is used the https://learn.microsoft.com/en-us/azure/storage/common/customer-managed-keys-overview
-   */
-  etcdEncryptionSetId?: string;
```

Comment:
etcd customer provided keys information will be based on vault references and not on azure disk encryption sets. Therefore we remove this.